docs/how-to-guides/chat-with-a-flow/index.md

```diff
@@ -0,0 +1,43 @@
+# Chat with a flow
+
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../faq.md#stable-vs-experimental).
+:::
+
+Prompt flow provides the chat window feature to facilitate an interactive chat experience within a local environment. 
+You can engage in conversation with the flow and view its responses directly within the chat window.
+
+## Initiating a chat window
+There are two methods to open a chat window: executing the prompt flow CLI command or clicking the 
+`Open test chat page` button when viewing a flow YAML file in the Prompt flow VS Code extension. To enable this 
+feature, you need to run `pf config set enable_internal_features="true"`.
```

Comment:
don't say this. Let's merge the PR after we make the feature graduate from internal_feature